### ISSUE TITLE
[ENH] - Start on JLab card should point to spawner options page

### ISSUE THREAD

**Issue by kcpevey** [nebari-dev/jhub-apps#88]
### Feature description

Currently, on the Jupyterlab card, the `Start` menu item makes an assumption on the server type and starts one for you without input. 

<img width="361" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/15659110/ae8dafec-807e-4a9b-be5d-a4ba713ffa01">

Recommendation: it should point to the spawner options page, just like clicking on the name `JupyterLab` on the card does. 

### Value and/or benefit

- Users will get the same experience via both routes of starting the Jupyterlab server. 
- Experienced users who expect to be able to choose a server will not be confused.

**Comment by aktech** [comment]
Good catch, I would suggest using the same interface for UX consistency.

Something like this basically:

<img width="1512" alt="Edit JupyterLab" src="https://github.com/nebari-dev/jhub-apps/assets/5647941/75cd2e78-b290-409e-a59b-6953943afe30">

Just disabling everything else except spawner profiles and (maybe conda environment)).
